webapp/src/components/EditEstatePage/EditEstateMetadata/EditEstateMetadataForm/EditEstateMetadataForm.js

```diff
@@ -44,6 +50,7 @@ export default class EditEstateMetadataForm extends React.PureComponent {
         name: event.target.value
       }
     })
+    this.setState({ formErrors: [] })
```

Comment:
Is it necessary to update the state of an unmounted component?

Reply:
We need to clean the errors if the user change the fields